javaslang/src/main/java/javaslang/collection/Multimaps.java

```diff
@@ -161,10 +185,30 @@ V getValue(Object[] pair) {
         }
     }
 
+    /**
```

Comment:
This one is for developers. It may help understanding the code. But it could also a single-line comment...